README.md

```diff
@@ -1,85 +1,35 @@
-<a name="readme-top"></a>
-
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
-<!-- TABLE OF CONTENTS -->
-
 # 📗 Table of Contents
 
-- [📖 About the Project](#about-project)
-  - [🛠 Built With](#built-with)
-    - [Tech Stack](#tech-stack)
-    - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
-- [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
-  - [Prerequisites](#prerequisites)
-  - [Install](#install)
-  - [Usage](#usage)
-  - [Run tests](#run-tests)
-  - [Deployment](#triangular_flag_on_post-deployment)
-- [👥 Authors](#authors)
-- [🔭 Future Features](#future-features)
-- [🤝 Contributing](#contributing)
-- [⭐️ Show your support](#support)
-- [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
-- [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+- [📗 Table of Contents](#-table-of-contents)
```

Comment:
- [ ] You didn't include `config/database.yml`.  Without this file, it is impossible to create the database. I had to create my own, but we don't want your users to pass through that hassle.  Since this project is for educational purposes, kindly include it, it is safe.

Or, you could make a local `.env` file in the root of your project and like to it from `database.yml`.  All your sensitive info can be in that file. 👍 

- [ ] Please, include instructions about how to set up the database with `rails db:create, rails db:migrate and rails db:seed`.
- [ ] There are no instructions in your readme about how to run the tests.  Kindly include those instructions.  I suggest `rake rswag`:
![image](https://user-images.githubusercontent.com/57421823/206009460-d8d0891c-0137-4ec6-9ada-a0ee45ef67a3.png)